[databricks][ccm] Add instructions to grant service principal access (#30863)
DataDog/documentation: content/en/cloud_cost_management/setup/saas_costs.md

{{% tab "Databricks" %}}

1. Navigate to the [Databricks integration tile][101] in Datadog and click **Configure**.
2. Enter the workspace name, url, client ID, and client secret corresponding to your Databricks service principal.
3. Under the **Select products to set up integration** section, click the toggle for each account to enable Databricks `Cloud Cost Management`.
4. Enter a `System Tables SQL Warehouse ID` corresponding to your Databricks instance's warehouse to query for system table billing data.
5. Click **Save Databricks Workspace**.

> Your service principal requires read access to the schemas within your catalog.

Reviewer: ```suggestion
Your service principal requires read access to the [system tables](https://docs.databricks.com/aws/en/admin/system-tables/) within Unity Catalog.
```

> Your service principal will require read access to the schemas within your catalog.

Reviewer 2: ```suggestion
Your service principal requires read access to the schemas within your catalog.
```

```sql

> GRANT USE CATALOG ON CATALOG <CATALOG> TO <service_principal>;

Reviewer: ```suggestion
GRANT USE CATALOG ON CATALOG system TO <service_principal>;
```

> GRANT USE SCHEMA ON CATALOG <CATALOG> TO <service_principal>;

Reviewer: ```suggestion
GRANT USE SCHEMA ON CATALOG system TO <service_principal>;
```

> GRANT SELECT ON CATALOG <CATALOG> TO <service_principal>;

Reviewer: ```suggestion
GRANT SELECT ON CATALOG system TO <service_principal>;
```
